<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dataLoggerWindow</class>
 <widget class="QWidget" name="dataLoggerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>309</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="connGroupBox">
     <property name="title">
      <string>Connection settings</string>
     </property>
     <layout class="QGridLayout">
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="1">
       <widget class="QComboBox" name="sourceTypeCombo"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="textLabel4">
        <property name="text">
         <string>&amp;Username:</string>
        </property>
        <property name="buddy">
         <cstring>editUsername</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel2">
        <property name="text">
         <string>D&amp;river</string>
        </property>
        <property name="buddy">
         <cstring>sourceTypeCombo</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="editDatabase"/>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="portSpinBox">
        <property name="specialValueText">
         <string>Default</string>
        </property>
        <property name="minimum">
         <number>-1</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="textLabel3">
        <property name="text">
         <string>Database Name:</string>
        </property>
        <property name="buddy">
         <cstring>editDatabase</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="editPassword">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="editUsername"/>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="editHostname"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="textLabel5">
        <property name="text">
         <string>&amp;Hostname:</string>
        </property>
        <property name="buddy">
         <cstring>editHostname</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="textLabel5_2">
        <property name="text">
         <string>P&amp;ort:</string>
        </property>
        <property name="buddy">
         <cstring>portSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="textLabel4_2">
        <property name="text">
         <string>&amp;Password:</string>
        </property>
        <property name="buddy">
         <cstring>editPassword</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="historicalSetupBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Setup Log Source</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
